### PR TITLE
Linux Package Create

### PR DESCRIPTION
**Pull request by khkh-ms**
### Issue describing the changes in this PR

resolves #issue_for_this_pr

### Pull request checklist

* [ ] My changes **do not** require documentation changes
  * [ ] Otherwise: Documentation issue linked to PR
* [ ] My changes **do not** need to be backported to a previous version
  * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [ ] I have added all required tests (Unit tests, E2E tests)

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -284,4 +285,70 @@ jobs:
...
       IntegrationTestsStorageAccountKey: $(IntegrationTestsStorageAccountKey)
     displayName: 'Upload Core Tools build to the storage account for Integration Tests'
-    condition: eq(variables.IsIntegrationBuild, 'true')
\ No newline at end of file
+    condition: eq(variables.IsIntegrationBuild, 'true')
+
+- job: LinuxPackage
+  condition: ne(variables['LinuxPackageBuildTag'], '')
+  timeoutInMinutes: "90"
+  pool:
```
Comment (arroyc) [inline azure-pipelines.yml:293]: i think it should be

pool:
    name: '1ES-Hosted-AzFunc'
    demands:
      - ImageOverride -equals MMS2022TLS

Comment (arroyc) [reply azure-pipelines.yml:293]: refer line 19/20

File: azure-pipelines.yml [hunk 2]
```diff
@@ -284,4 +285,70 @@ jobs:
...
+    vmImage: 'ubuntu-latest'
+  steps:
+  # Bash v3
+  # Run a Bash script on macOS, Linux, or Windows.
+  - task: Bash@3
+    inputs:
+      targetType: 'inline'  # Specify 'filePath' if you want to use an external script file.
+      script: |
```
Comment (arroyc) [inline azure-pipelines.yml:301]: can we move this to an individual script and run the script as a task instead

Comment (khkh-ms) [reply azure-pipelines.yml:301]: We will address it as part of other ticket.

File: azure-pipelines.yml (hunk 1 above)
Comment (khkh-ms) [reply azure-pipelines.yml:293]: We will address it as part of other ticket.

**Comment by arroyc** [comment]
create a seperate workitem to workone the pipeline improvement

**Review by arroyc — APPROVED** [review]

**Pull request merged**